e2e_tests/tests/experiment/test_delete.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+import subprocess
+import time
+
+import pytest
+
+from determined.common import api
+from tests import config as conf
+from tests import experiment as exp
+
+
+@pytest.mark.e2e_cpu
+def test_delete_experiment_removes_tensorboard_files() -> None:
+    """
+    Start a random experiment, delete the experiment and verify that TensorBoard files are deleted.
+    """
+    config_obj = conf.load_config(conf.tutorials_path("mnist_pytorch/const.yaml"))
+    experiment_id = exp.run_basic_test_with_temp_config(
+        config_obj, conf.tutorials_path("mnist_pytorch"), 1
+    )
+
+    command = ["det", "e", "delete", str(experiment_id), "--yes"]
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+    ticks = 120
+    for i in range(ticks):
```

Comment:
there's a utility function for this: `tests.experiment.experiment.wait_for_experiment_state`

Reply:
I initially tried to do this, it seems like `wait_for_experiment_state(exp_id, experimentv1State.DELETED, max_wait_secs=120)` does not work for DELETED experiements as I don't think they are in the db, so it returns an error. This is an existing workaround for this in another test: https://github.com/determined-ai/determined/blob/4d3eaab29902c988026f7c5634ea5bce9e79b33a/e2e_tests/tests/cluster/test_checkpoints.py#L383